.buildkite/pipeline-utils/ci-stats/pick_test_group_run_order.ts

```diff
@@ -129,7 +129,12 @@ function getEnabledFtrConfigs(patterns?: string[], solutions?: string[]) {
 
   const mappedSolutions = solutions?.map((s) => (s === 'observability' ? 'oblt' : s));
   for (const manifestRelPath of ALL_FTR_MANIFEST_REL_PATHS) {
-    if (mappedSolutions && !mappedSolutions.some((s) => manifestRelPath.includes(`ftr_${s}_`))) {
+    if (
+      (mappedSolutions && !mappedSolutions.some((s) => manifestRelPath.includes(`ftr_${s}_`))) ||
+      // When applying the solution filter, still allow platform tests
+      manifestRelPath.includes('ftr_platform_') ||
```

Comment:
Something looks a little off on the recent run, like all the platform tests are skipped.

Reply:
oh! I should have negated these. Addressed in https://github.com/elastic/kibana/pull/235929/commits/66f134c9b74c249f93b4f74f0195b39beaf750f5